jest-unit.config.js

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line
```

Comment:
A particular unit test would fail without this code.

Reply:
It can be useful to disable the specific rule, so that others know what is being worked around and additional violations aren't introduced.
```suggestion
// eslint-disable-next-line rule-name
```

Reply:
Thanks for the suggestion, I'll look into adding the specific rule here and in a similar file for E2E tests. Just to be clear, the comment about unit tests failing are not referring to the particular line, but to the addition of`transformIgnorePatterns`.